### ISSUE TITLE
/Bmactivity 

### ISSUE THREAD

**Issue by Nrevi** [BanManagement/BanManager#732]
So... I have a staff member named Firewalllol. There's multiple players named Firewalllol
http://byazo.com/41p8_05_06_2017.png

Therefore I can't really do /bmactivity 1y Firewalllol. Is there any way to delete one of the profiles? Or any way to do /bmactivity 1y <uuid> ?

**Comment by confuser** [comment]
`/bmactivity 1y Firewalllol` should work.

**Comment by Nrevi** [comment]
I already did that though.. This is what it says when I type it: http://byazo.com/61Ha_05_06_2017.gif

**Comment by confuser** [comment]
Yes, you can do `/bmactivity 1y` but it will probably return a lot of results. It doesn't support multiple players with the same name like /bminfo does.

The additional player should have attempted to resolve itself by retrieving the new name.

The real fix would be for the command to accept a uuid, and support multiple players with the same name.

In the mean time, you could perform a manual lookup yourself by UUID and update the affected player, within the players table.

**Comment by confuser** [comment]
NB: Documentation and plugin.yml needs updating to reflect new command options
